### DISCUSSION TITLE
Some errors

### DISCUSSION THREAD

**Discussion by liahsuum** [jan-vandenberg/cruddiy#59] [General]
Hello:
I found your project and I like it. But, the same has 2 errors and I like your answer.
When display the information:
Notice: Undefined index: HTTPS in C:\xampp\htdocs\CRUD_DIY\app\autores-index.php on line 51
When search:
ERROR: Could not able to execute SELECT * FROM materias WHERE CONCAT_WS (descriptores) LIKE '%historia%' ORDER BY id desc LIMIT 0, 10. Incorrect parameter count in the call to native function 'CONCAT_WS'
Thanks!

**Comment by yvaha** [comment]
The HTTPS thing is a warning. It appears in XAMPP. It is not essential for the working of the application.

The CONCAT_WS function requires at least two parameters. It can be replaced by the CONCAT function which does not have this flaw, or it can be solved by adding an empty string as a parameter e.g. CONCAT_WS("", fieldname_in_database).

Other error I detected is in the links generated for the headers of the columns.
The parameter sort is passed twice to the URL, thus compromising the function of the sorting.
In the section //Column sort order, the default sorting order is set to desc. In most cases people want to see a table from the first records. 

The text **ERROR: Could not able to execute** should read **ERROR: _Was_ not able to execute**

**Comment by jan-vandenberg** [comment]
Good points!

Feel free to send a PR, I will pull them in Cruddiy.

**Comment by germain-italic** [comment]
Hi @yvaha i've created issue #89 referring this problem, and proposed a PR in [this commit](https://github.com/jan-vandenberg/cruddiy/commit/64109d240c370e858a4cee4138c11021b7cce0cc) to address the issue.
